From c973f2910322a2b6cfae210730bfd6d3d8259e01 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Sun, 1 Mar 2020 10:41:30 +0200
Subject: [PATCH] meson: Disable unused dbusmock check as tests are commented
 out

This avoids issues with meson-0.53 like commit 1853567d79024, but
by just dropping the check, because the 0.13.1 release had tests
commented out fully anyhow.
---
 meson.build | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/meson.build b/meson.build
index 8d09149e..8f52bb7e 100644
--- a/meson.build
+++ b/meson.build
@@ -123,10 +123,6 @@ endif
 
 if bluez_backend_enabled
   libebook_dep = dependency('libebook-1.2', version: '>=' + min_eds_version)
-  # Needed for the BlueZ tests
-  pymod = import('python')
-  py_installation = pymod.find_installation('python3')
-  python_dbusmock = py_installation.dependency('dbusmock')
 endif
 
 if import_tool_enabled
-- 
2.20.1

